tests/test_backend.py

```diff
@@ -51,6 +51,10 @@ async def test_set_exist(cache):
     assert not await cache.set("key2", b"value", exist=False)
 
 
+async def test_set_many(cache):
+    assert await cache.set_many({"key1": "value1", "key2": "value2"})
```

Comment:
plz remake this test .
https://github.com/Krukov/cashews/pull/88/checks#step:6:233

I think it will be better to do `set_many` and then check that values was set